fiftyone/migrations/revisions/v1_4_0.py

```
@@ -0,0 +1,62 @@
+    bulk_updates = [
+        UpdateOne(
+            {"_id": op["_id"]},
+            {"$set": {target_field: op[source_field], source_field: None}},
+        )
+        for op in ops_to_update
+    ]
+
+    try:
+        result = db.delegated_ops.bulk_write(bulk_updates)
+        return result
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)